frontend/src/views/HomePage/HomePage.scss

```diff
@@ -0,0 +1,53 @@
+body{
+    background-color: #48CAE4;
+}
+
+.navList{
+    margin-left: 10%;
+    display: flex;
+    flex-direction: column;
+}
+
+.daytime{
+position: absolute;
+top: 0px;
+right: 10px;
+}
+
+
+.title h1{
+    font-family: 'Roboto Condensed', sans-serif;
+    font-weight: 900;
+    font-size: 8rem;
+    margin-bottom: 0;
+    margin-top: 40%;
+    color: #102A71;
+}
+
+.title h2{
+    font-family: 'Roboto', sans-serif;
+    font-weight: 900;
+    font-size: 2.5rem;
+    margin-top: 0;
+    letter-spacing: 0.2rem;
+    color:#0096C7 ;
+    width: 100%;
+}
+
+.homeContainer{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+
+.aidKit{
+    box-sizing: border-box;
+    width: 20vh;
+}
+
+img.aidKit {
+    margin-top: 60vh;
+    margin-right: 10vh;
+    margin-left: 0;
+
+}
```

Comment:
Let's follow the CSS naming convention from App.scss (use hypen-separate-names instead of camelCase) eg. aid-kit, home-container

One last thing - those styles such (title h1) cause problems when the rest of the code is merged - someone can have a a h1 inside title as well. Try to create more specific selectors (eg. .home-container .title h1).